### PR TITLE
New version: Ablaze.Floorp version 11.9.0

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/137657)

### CODE REVIEW

**Comment by stephengillie** [comment]
Manual Validation ended with: 
image

**Comment by stephengillie** [comment]
Hi @Exorcism0666,

The installation isn't unattended. Is there an installer switch to have the package install silently?

(Automated response - build 695.)

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

File: manifests/a/Ablaze/Floorp/11.9.0/Ablaze.Floorp.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Created with komac v2.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Ablaze.Floorp
+PackageVersion: 11.9.0
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.17763.0
+InstallerType: exe
```
Comment (Hibi-10000) [inline manifests/a/Ablaze/Floorp/11.9.0/Ablaze.Floorp.installer.yaml:10]:
```suggestion
InstallerType: nullsoft
```

File: manifests/a/Ablaze/Floorp/11.9.0/Ablaze.Floorp.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+PackageLocale: en-US
+Publisher: Ablaze
+PublisherUrl: https://ablaze.one/
+PublisherSupportUrl: https://support.ablaze.one/articles/floorp/
+PrivacyUrl: https://docs.ablaze.one/floorp_privacy_policy/
+Author: Ablaze
+PackageName: Ablaze Floorp
+PackageUrl: https://floorp.app/en/
+License: Mozilla Public License Version 2.0
+LicenseUrl: https://github.com/Floorp-Projects/Floorp/blob/HEAD/LICENSE
```
Comment (Hibi-10000) [inline manifests/a/Ablaze/Floorp/11.9.0/Ablaze.Floorp.locale.en-US.yaml:15]:
```suggestion
LicenseUrl: https://github.com/Floorp-Projects/Floorp/blob/ESR115/LICENSE
```

File: manifests/a/Ablaze/Floorp/11.9.0/Ablaze.Floorp.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  🌟 What’s new!
+  Manage Your Web Apps on preferences page!
+  If the web apps have compatibility, You can manage all web apps on preferences page. Check all web apps or uninstall them.imageAdd some preferences to relate to PWA on preferences page.
+  Now, you can disable all toolbar on PWA window.imageℹ️ Changed
+  Now, “Back” & “Reload” buttons available on PWA window.
+  imageIf “Show Workspace Name on Tab Bar’s Workspace Button” is enabled, Workspaces can run without Workspaces toolbar button.
+  image❌ Fixed
+  Fix invisible vertical tab bar background, when user enable collapse vertical tab bar.
+  Fix In vertical tabs, when dragging out tabs to create a window, the area to be dragged was too large.
+ReleaseNotesUrl: https://github.com/Floorp-Projects/Floorp/releases/tag/v11.9.0
```
Comment (Hibi-10000) [inline manifests/a/Ablaze/Floorp/11.9.0/Ablaze.Floorp.locale.en-US.yaml:38]:
```suggestion
ReleaseNotesUrl: https://blog.ablaze.one/3926/
```

File: manifests/a/Ablaze/Floorp/11.9.0/Ablaze.Floorp.locale.ja-JP.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+- browser
+- cross-platform
+- firefox-fork
+- foss
+- gecko
+- internet-browser
+- open-source
+- web
+- web-browser
+ReleaseNotesUrl: https://blog.ablaze.one/3904/
```
Comment (Hibi-10000) [inline manifests/a/Ablaze/Floorp/11.9.0/Ablaze.Floorp.locale.ja-JP.yaml:27]:
```suggestion
ReleaseNotesUrl: https://blog.ablaze.one/3935/
```

**Review by Hibi-10000 — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**